XCUITests/ThirdPartySearchTest.swift

```diff
@@ -102,7 +96,16 @@ class ThirdPartySearchTest: BaseTestCase {
         XCTAssertFalse(app.scrollViews.otherElements.buttons["Mozilla Engine search"].exists)
     }
     
-    private func DismissSearchScreen() {
+    private func addCustomSearchEngine() {
+        navigator.performAction(Action.AddCustomSearchEngine)
+        waitForExistence(app.buttons["customEngineSaveButton"], timeout: 3)
+        app.buttons["customEngineSaveButton"].tap()
+        // Workaround for iOS14 need to wait for those elements and tap again
+        waitForExistence(app.navigationBars["Add Search Engine"], timeout: 3)
```

Comment:
Does this need a wrapper iOS version check?

Reply:
Yup you are totally right, I will fix that and merge, thanks!